build-in-container/mariner-docker-builder.sh

```diff
@@ -21,7 +21,7 @@ help() {
 
     Optional arguments:
     --mariner_dir             directory to use for Mariner artifacts (SPECS, toolkit, ..). Default is the current directory
-    --RPM_repo_file           Path to a custom repo file.
+    --RPM_repo_file           Path to a custom repo file (must end in .repo). Provide multiple filepaths with comma (,) as delimiter.
```

Comment:
Should we do a validation check on the user-supplied parameter to enforce that a `.repo` extension is used? Not quite sure if we can check any further, but this could inform the user at invocation time

Reply:
Given these are all being cat'd into a single file the tool controls this is actually not super important in hindsight.

Reply:
Removed copying the contents into a single file. Now, I am copying the user provided custom-repo.repo file into the container. So, this check becomes necessary